source/mainnet/net/guides/change-account-name.rst

```diff
@@ -7,13 +7,25 @@
 Change account name
 ===================
 
-#. Tap on the account you want to rename.
+.. dropdown:: |mw-gen2|
 
-#. In the Account details screen, tap the hamburger menu |hamburger|.
+    #. Tap on the account you want to rename.
 
-#. Tap **Change account name**.
+    #. In the Account details screen, tap |acct-settings|.
 
-#. Change the name and click |save| to save the change.
+    #. Tap **Change account name**.
+
+    #. Change the name and click |save| to save the change.
+
+.. dropdown:: |mw-gen1|
```

Comment:
It used to be because we have had this here almost the entire time I've been at Concordium. However, I can see that the option is no longer in the hamburger menu.